README.md

```diff
@@ -520,7 +521,8 @@ IEEE Conf. Computer Vision and Pattern Recognition (CVPR), 2019. [PDF](https://a
 IEEE Conf. Computer Vision and Pattern Recognition (CVPR), 2019.
 - [Gehrig et. al. arXiv 2019](#Gehrig19arxiv),  
 *End-to-End Learning of Representations for Asynchronous Event-Based Data*.
-
+- [Zhu et. al. CVPR 2019](#Zhu19cvpr),  
```

Comment:
This one is not needed. I wrote the paper in this same section, a few lines above.

Reply:
Ah I didn't see that it'd been added already. I've removed my addition.